tasks/deploy_success.yml

```diff
@@ -4,14 +4,7 @@
     token: "{{ slack_webhook_token }}"
     channel: slack_webhook_channel
     attachments:
-      - text: "Deployment completed successfully"
+      - text: "*{{ lookup('pipe', 'git config user.name') }}* successfully deployed {{ site }} to *<https://{{ project.site_hosts | map(attribute='canonical') | first }}|{{ env }}>* [<{{ slack_git_repo_url }}/commit/{{ git_clone.after }}|deployed commit>]"
```

Comment:
How about git remotes that dont provide a web interface?

We need to make this message support multiple git remotes:
1. no interface, i,e: before this PR
1. GitHub
1. GitLab
1. Bitbucket

Reply:
Rather than using `slack_git_repo_url`, determine which git remote is in used by prasing https://github.com/roots/trellis/blob/f11a7cefbc978831d1ff6b4156298e618ff2ff80/group_vars/production/wordpress_sites.yml#L12 is a better choice.

Reply:
As far as I know (definitely need to look it up) `git config` isn't tied to GitHub specifically, so the username should work for all in theory - will find out for sure when I'm back on Monday.

Will also look into using the defined site repo as basis for the commit link / web interface (if one is available, maybe only link if GitHub for now?)

Reply:
> git config isn't tied to GitHub specifically, so the username should work for all in theory

Correct.

> Will also look into using the defined site repo as basis for the commit link / web interface (if one is available, maybe only link if GitHub for now?)

Just these 2 are okay:
1. unknown git server, i,e: before this PR
2. GitHub

Reply:
Should support GitHub + non-Github repo's now. Not v. clued up on Ansible so might be a bit of a weird solution...